frontend/src/components/fragments/emotes/emote-dropzone.component.tsx

```diff
@@ -0,0 +1,71 @@
+import { bind } from 'decko';
+import React, { Component } from 'react';
+import 'react-dropzone-uploader/dist/styles.css';
+import Dropzone from 'react-dropzone-uploader';
+
```

Comment:
In this case, I used "any" for the props so I didn't have to take the time (right now) to define the props it uses.  I can/will go back later if we decide to keep the typescript path and move all these over to strict types.